tests/test_endpoint.py

```diff
@@ -65,16 +66,34 @@ def test_api_can_add_plant_to_garden(self):
         self.assertIn("{} 00:00:00 GMT".format(parsed_harvest_date), str(res.data))
 
     def test_api_can_add_plant_to_garden_with_no_harvest_date(self):
+        garden = Garden(id=1)
         zeke = Plant(plant_type='Cherry Tomato',image='jim_photo.jpg',lighting='Full Sun',water_frequency=3,harvest_time=50,root_depth=12,annual="Annual")
         dan = Plant(plant_type='Cactus',image='cactus_dan.jpg',lighting='Full Sun',water_frequency=7,harvest_time=None,root_depth=8,annual="Annual")
-        db.session.add_all([zeke, dan])
+        db.session.add_all([zeke, dan, garden])
         db.session.commit()
 
         res = self.client().post('/api/v1/garden?plant_id={}&plant_name=Marjorie'.format(dan.id))
         self.assertEqual(res.status_code, 201)
         self.assertIn("Marjorie", str(res.data))
         self.assertIn("null", str(res.data))
 
+    def test_api_can_return_all_plants_in_garden(self):
+        garden = Garden(id=1)
+        zeke = Plant(plant_type='Cherry Tomato',image='jim_photo.jpg',lighting='Full Sun',water_frequency=3,harvest_time=50,root_depth=12,annual="Annual")
+        dan = Plant(plant_type='Cactus',image='cactus_dan.jpg',lighting='Full Sun',water_frequency=7,harvest_time=None,root_depth=8,annual="Annual")
+        agatha = Plant(plant_type='Roma Tomato',image='agatha_photo.jpg',lighting='Full Sun',water_frequency=2,harvest_time=60,root_depth=12,annual="Annual")
+        db.session.add_all([zeke, dan, agatha, garden])
+        db.session.commit()
+
+        res = self.client().post('/api/v1/garden?plant_id={}&plant_name=Ezekiel'.format(zeke.id))
+        res = self.client().post('/api/v1/garden?plant_id={}&plant_name=Dan'.format(dan.id))
+
+        res = self.client().get('/api/v1/garden')
+        self.assertEqual(res.status_code, 200)
+        self.assertIn("Ezekiel", str(res.data))
+        self.assertIn("Dan", str(res.data))
+        self.assertNotIn("Agatha", str(res.data))
```

Comment:
Testing will be a big 'come back to this' later.  I'm sure that there are some edge cases we should be testing for.